packages/core/src/optimize/development.ts

```diff
@@ -73,12 +73,18 @@ export default async (app: express.Application): Promise<void> => {
     await sleep(100);
   }
 
-  const proxyMiddleware = proxy({
+  const defaultProxyConfig = {
     target: `http://localhost:${devServerPort}`,
     ws: true,
     changeOrigin: true,
     logLevel: 'error',
-  });
+  } as proxy.Config
+
+  const proxyConfig = config.proxyMiddleware
+    ? config.proxyMiddleware(defaultProxyConfig)
+    : defaultProxyConfig
```

Comment:
Could you update README?

https://github.com/saltyshiomix/react-ssr#configuration-ssrconfigjs
https://github.com/saltyshiomix/react-ssr/tree/master/packages/express#configuration-ssrconfigjs
https://github.com/saltyshiomix/react-ssr/tree/master/packages/nestjs-express#configuration-ssrconfigjs